docs/community/contributing.md

```diff
@@ -0,0 +1,69 @@
+## Contributing to OpenMMLab
+
```

Comment:
Downstream OpenMMLab projects all rely on the CONTRIBUTING.md at https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md